objectRecognitionSource/src/main/java/io/vantiq/extsrc/objectRecognition/neuralNet/YoloProcessor.java

```diff
@@ -227,6 +241,29 @@ private void setup(Map<String, ?> neuralNet, String sourceName, String modelDire
            // Flag to mark that we should not save images
            imageUtil.saveImage = false;
        }
+       
+       // Checking if pre cropping was specified in config
+       if (neuralNet.get("preCrop") instanceof Map) {
```

Comment:
change name to `cropBeforeAnalysis` or something like that.

Variable name is fine, but like to have better name for configuration.